src/gmpy2_convert_gmp.c

```diff
@@ -222,10 +222,18 @@ GMPy_PyLong_From_MPZ(MPZ_Object *obj, CTXT_Object *context)
     while ((size>0) && (result->ob_digit[size-1] == 0)) {
         size--;
     }
+#if PY_VERSION_HEX >= 0x030900A0
```

Comment:
```suggestion
#if PY_VERSION_HEX >= 0x030900A4
```

This from https://docs.python.org/3.10/whatsnew/3.10.html

Reply:
Good catch.  I've updated the commit as suggested.